server/Dockerfile

```diff
@@ -145,59 +101,67 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim"
         python3-setuptools \
         python3-pip \
         # PHP Requirements
-        php \
-        php-apcu \
-        php-curl \
-        php-xml \
-        php-intl \
-        php-bcmath \
-        php-mbstring \
-        php-mysql \
-        php-redis \
-        php-gd \
-        php-fpm \
-        php-zip \
+        php7.4 \
+        php7.4-apcu \
+        php7.4-curl \
+        php7.4-xml \
+        php7.4-intl \
+        php7.4-bcmath \
+        php7.4-mbstring \
+        php7.4-mysql \
+        php7.4-redis \
+        php7.4-gd \
+        php7.4-fpm \
+        php7.4-zip \
         librdkafka1 \
         libbrotli1 \
-        libsimdjson5 \
+        libsimdjson9 \
         # Unsure we need these
         zip unzip \
         # Require for advanced an unattended configuration
         curl jq \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
-    # Download MISP using git in the /var/www/ directory.
-    RUN if [ ! -z ${MISP_COMMIT} ]; then \
-        git clone https://github.com/MISP/MISP.git /var/www/MISP && cd /var/www/MISP && git checkout ${MISP_COMMIT}; \
-        else git clone --branch ${MISP_TAG} --depth 1 https://github.com/MISP/MISP.git /var/www/MISP; fi
-    RUN cd /var/www/MISP; git submodule update --init --recursive .; cd /var/www/MISP/app; \
-        # Remove some old and broken links that pollute the log files
-        rm -rf /var/www/MISP/INSTALL/old
+    # Download MISP using git in the /var/www/ directory. Remove unnecessary items.
+    RUN <<-EOF
+            if [ ! -z "${MISP_COMMIT}" ]; then
+                git clone https://github.com/MISP/MISP.git /var/www/MISP && cd /var/www/MISP && git checkout "${MISP_COMMIT}"
+            else
+                git clone --branch "${MISP_TAG}" --depth 1 https://github.com/MISP/MISP.git /var/www/MISP
+            fi
+            cd /var/www/MISP || exit; git -c submodule."PyMISP".update=none submodule update --init --recursive .
```

Comment:
Skipping grabbing the PyMISP submodule since we've already fetched it via wheel

Reply:
Refer to our conversation

Reply:
Resolving this thread so we can discuss it in [this one](https://github.com/ostefano/docker-misp/pull/25#pullrequestreview-1608527366)